x-pack/plugins/observability_solution/apm/server/routes/entities/get_entity_latest_services.ts

```diff
@@ -54,7 +55,7 @@ export async function getEntityLatestServices({
         },
       },
     })
-  ).hits.hits.map((hit) => hit._source);
+  ).hits.hits.map((hit) => normalizeFields(hit?.fields ?? {}) as unknown as EntityLatestServiceRaw);
```

Comment:
need to replace with an explicit mapping, right?

Reply:
Thanks! It's on our list to replace this function everywhere. We discussed that with @miloszmarcinkowski  and I am wondering if we want to have this change for the services that are coming from EEM - so the view is still in tech preview and idk if we want to have the changes there as this should not affect open telemetry "classic" view. 
This is for the "new" view when this button is clicked: 
<img width="367" alt="image" src="https://github.com/user-attachments/assets/ff18b90d-dfa1-442f-aaf7-422042d8c222">

Do we want to do this change there as well?

Reply:
Yes please. In fact, any new feature being developed in the UI should be from now on "OTel-native" (i.e. not using `_source`) from the beginning.

Reply:
I reverted the changes and it won't affect the services with Otel ( I see no errors when I enabled the new view )